src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
+Dart is a [garbage-collected][] language,
+which means that any Dart object
+that isn't referenced by at least one other object
+will eventually be disposed by the garbage collector. 
+This default behavior may not be desireable in
+some scenarios involving native resources or 
+if the target object cannot be sub-classed.
+
+A [WeakReference][]
+stores a reference to the target object
+without increasing the reference count
+considered by the garbage collector.
+Another option is to use an [Expando][]
+to add properties to an object
+without affecting how they are garbage-collected.
```

Comment:
I think this should only be [hyphenated](https://developers.google.com/style/hyphens?hl=en) when used before a noun:

```suggestion
without affecting how they are garbage collected.
```

Reply:
+1. Should be:
without affecting how it's garbage collected.